### PR TITLE
Moves playsound_local() to mob and gets rid of a useless proc.

### PR DESCRIPTION
**Pull request by ghost**
Playsound_local was previously on atom/movable, even though a chair can't hear or whatever. Also, there was a duplicate of the proc for no reason called playsound_direct, which was unused!

### CODE REVIEW

**Comment by Really-Good-Soda-Flavor** [comment]
No, MMIs and stuff have a mob variable.

**Comment by Cyberboss** [comment]
I just added `playsound_direct` last month. It's for playing sounds without 3D positioning. `playsound_local` needs to remain on atom/movable because of possession and other memes.

**Comment by Really-Good-Soda-Flavor** [comment]
Well, it is useless at the moment, and playsound_local clearly does not have to remain on atom as evidenced by its move.

**Comment by Cyberboss** [comment]
It's better to have an extra proc than to require every sound that wants to play directly to a mob use `<<` which we're trying to move away from, or playsound_local with a get_turf(src)

**Comment by Really-Good-Soda-Flavor** [comment]
> It's better to have an extra proc than to require every sound that wants to play directly to a mob use << which we're trying to move away from, or playsound_local with a get_turf(src)

> You can literally just set the mob as the location in playsound_local, and that is all playsound_direct does. There is no need for another proc.

**Comment by duncathan** [comment]
if an admin is possessing a chair and someone calls the chair's `playsound_local()`, does the admin hear the sound? if yes then don't move the definition
:+1: to removing the redundant proc

File: code/game/sound.dm [hunk 1]
```diff
@@ -80,11 +80,6 @@
```
Comment (duncathan) [inline code/game/sound.dm]: while you're in here, there's a line `S.falloff = (falloff ? falloff : FALLOFF_SOUNDS)` that should be replaced with `S.falloff = falloff || FALLOFF_SOUNDS`

Comment (duncathan) [inline code/game/sound.dm]: it's line 79, right above this line i commented on

**Pull request merged**